modules/oc-mirror-oci-format.adoc

```diff
@@ -15,7 +15,7 @@ The local catalog and its contents are mirrored to your target mirror registry b
 ====
 When mirroring local OCI catalogs, any {product-title} releases or additional images that you want to mirror along with the local OCI-formatted catalog must be pulled from a registry.
 
-You cannot mirror OCI catalogs along with an oc-mirror image set file on disk.
+You cannot mirror OCI catalogs along with an oc-mirror plugin v1 image set file on disk.
```

Comment:
@zhouying7780 is this true ?

Reply:
Sorry , I can't understand what's the meaning here .

Reply:
@kasturinarra No, this is not ture, oc-mirror v1 also support to mirror OCI catalogs.